### DISCUSSION TITLE
I'm confuzed - Altz/Az vs Equatorial

### DISCUSSION THREAD

**Discussion by Thinker3932** [Stellarium/stellarium#3184] [General]
My SkyWatcher Star Adventurers (21 or GTi) only have two controls, RA or Dec.  Regardless of whether I'm in Alt/Az or Equatorial mode on Stellarium, it doesn't seem to be simulating my mount when I hit the up/down or left/right keys.  It does seem to match the behavior of what an Alt/Az mount would do (regardless of whether I am in Alt/Az or EQ mode), but that doesn't help.

For example, from North America, if I frame Polaris in photographic landscape orientation and then slew around to take something about 45 degrees high due south (as is shown in the attached photo), the camera can only be in photographic portrait orientation. Not only does Stellarium still insist my framing would be landscape, the up/down and left/right are clearly not RA and Dec, they stay alt/az.  

What am I missing?  I want to be able to simulate the POV that my camera would have on an EQ mount.  I want to be able to practice adjusting RA and Dec in Stellarium (or figure out which I would need to adjust in the real world when trying to slew to a target) but this program doesn't seem to have the ability to do that.

?? What am I missing?  https://drive.google.com/file/d/1nRbrWmXHxrnCikf78jGeLnr6GaRAudzb/view?usp=sharing

**Comment by JChase67** [comment]
(Oculars menu)  >Settings (wrench),  >Telescopes (tab)  >Equatorial Mount (checkbox).
